node/relay.go

```diff
@@ -0,0 +1,88 @@
+package node
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/signal"
+	"path/filepath"
+	"syscall"
+
+	"github.com/libp2p/go-libp2p/core/peer"
+	"go.uber.org/zap"
+
+	"github.com/spacemeshos/go-spacemesh/common/types"
+	"github.com/spacemeshos/go-spacemesh/config"
+	"github.com/spacemeshos/go-spacemesh/log"
+	"github.com/spacemeshos/go-spacemesh/metrics"
+	"github.com/spacemeshos/go-spacemesh/p2p"
+	"github.com/spacemeshos/go-spacemesh/p2p/handshake"
+)
+
+var relayAddrInfoCh chan peer.AddrInfo // used for testing
+
+func runRelay(ctx context.Context, cfg *config.Config) error {
+	ctx, cancel := signal.NotifyContext(ctx, os.Interrupt, syscall.SIGTERM)
+	defer cancel()
```

Comment:
Can this be factored out? the `node` command does the same thing and probably every other command should have this as well 🙂

Reply:
Rather hard to factor out without somewhat deeper refactoring of `App` stuff, I think. Hopefully `relay` command is only temporary need, so maybe not worth it